libdw-sys/build.rs

```diff
@@ -1,7 +1,7 @@
 extern crate pkg_config;
 
 fn main() {
-    assert!(pkg_config::target_supported());
+    assert!(pkg_config::Config::new().target_supported());
     if pkg_config::probe_library("libdw").is_err() {
```

Comment:
Let's make one `Config` and use it for both calls.
(`probe_library(name)` is just `Config::new().probe(name)`)

Reply:
Sorted!

Looks like the old method has been readded as a result of your comments: https://github.com/rust-lang/pkg-config-rs/pull/128

However, given it's marked as deprecated, this change is still probably worth having.